swap_meet/clothing.py

```diff
@@ -1,2 +1,17 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Clothing(Item):
+    def __init__(self, id = None, fabric = "Unknown", condition = 0, age = 0):
```

Comment:
[nit] I personally prefer this style, but PEP 8 says not to put the spaces for default params like this. 😭  `fabric="Unknown"`